### PR TITLE
Ensure co-commit works correctly with multiple co-authors

### PR DESCRIPTION
**Pull request by jul-sh**
I recommend browsing by commit. :) 

Not super happy with the current naming convention of the tests. Will amend these naming in another PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,19 +23,21 @@ const gatherInput = async () => {
     }
   )
 
+  console.log(args['--co-authors'])
```
Comment (Tom-Bonnike) [inline index.js]: Do you wanna leave that log here?

File: index.js [hunk 2]
```diff
@@ -55,11 +57,11 @@ const gatherInput = async () => {
 const writeCommit = async ({ message, coAuthors, otherArgs }) => {
   const coAuthoringLines = coAuthors.map(
     coAuthor =>
-      `-m "Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>"`
+      `Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>`
   )
-  const gitCommand = `git commit -m "${message}" ${coAuthoringLines.join(
-    ' '
-  )} ${otherArgs.join(' ')}`
+  const gitCommand = `git commit -m "${message}\n\n${coAuthoringLines.join(
```
Comment (Tom-Bonnike) [inline index.js]: I’m not sure if those `\n` behave the same on all environments, but hey, it’d be a problem with the co authoring lines anyway 😄

File: index.test.js [hunk 3]
```diff
@@ -40,7 +40,41 @@ describe('co-commit', () => {
...
+
+        if (!stdout.includes(expectedGitCommand))
+          return new Error('Does not output correct git command')
+      })
+      .code(0)
+      .end(done)
+  })
+
+  it('ouputs the correct git commit when commiting with multiple co-authors', done => {
+    nixt()
```
Comment (Tom-Bonnike) [inline index.test.js:54]: TIL about nixt, nice.

File: index.js (hunk 1 above)
Comment (jul-sh) [reply index.js]: Yes 😂

File: index.js (hunk 2 above)
Comment (juliettepretot) [reply index.js]: Pretty sure it's handled by Node.js, so should be fine :)

File: index.test.js (hunk 3 above)
Comment (jul-sh) [reply index.test.js:54]: Me likez, itz a noize tool

File: index.js (hunk 2 above)
Comment (juliettepretot) [reply index.js]: I'll try this on Windows rig tonight though, just to be sure. :)

Comment (jul-sh) [reply index.js]: Fair point, let's use https://nodejs.org/api/os.html#os_os_eol though, probs better

Comment (Tom-Bonnike) [reply index.js]: Ah yeah nice nice nice

File: index.js [hunk 4]
```diff
@@ -55,11 +56,12 @@ const gatherInput = async () => {
...
-      `-m "Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>"`
+      `Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>`
   )
-  const gitCommand = `git commit -m "${message}" ${coAuthoringLines.join(
-    ' '
-  )} ${otherArgs.join(' ')}`
+  const gitCommand = `git commit -m "${message +
+    os.EOL +
+    os.EOL +
+    coAuthoringLines.join(os.EOL)}" ${otherArgs.join(' ')}`
```
Comment (Tom-Bonnike) [inline index.js]: You could write it like
```js
const coAuthoringMessage = coAuthoringLines.join(os.EOL)
const gitCommand =
  `git commit -m "${message}" -m "${coAuthoringMessage}" ${otherArgs.join(' ')}`
```

which might be more readable.
Or if you don’t want to use the double `-m` flags,

```js
const coAuthoringMessage = [
  os.EOL,
  os.EOL,
  coAuthoringLines.join(os.EOL)
].join('')
const gitCommand =
  `git commit -m "${message + coAuthoringMessage}" ${otherArgs.join(' ')}`
```

**Review by MariiaPunda — APPROVED** [review]

**Review by Tom-Bonnike — APPROVED** [review]
But LGTM!

File: index.js (hunk 4 above)
Comment (jul-sh) [reply index.js]: The second option looks quite nice, let's use that. :D

**Pull request merged**